layouts/MainLayout.js

```diff
@@ -6,7 +6,14 @@ import Header from '../containers/header/Header';
 import Footer from '../containers/footer/Footer';
 
 
-const MainLayout = ({ className, children }) => (
+const PageContainer = styled('div')({
```

Comment:
Should use the material container component instead: https://material-ui.com/components/container/